ush/config_defaults.yaml

```diff
@@ -3052,4 +3054,5 @@ cpl_aqm_parm:
   DCOMINpt_src_dfv: "/path/to/point/source/base/directory"
 
   DCOMINairnow_dfv: "/path/to/airnow/obaservation/data"
-  AQM_AIRNOW_HIST_DIR: "/path/to/historical/airnow/data/dir"
+  COMINbicor: "/path/to/historical/airnow/data/dir"
+  COMOUTbicor: "/path/to/historical/airnow/data/dir"
```

Comment:
The descriptions for COMINbicor and COMOUTbicor ("/path/to/~") are the same each other. Please add a couple of words to distinguish between them.